src/components/interaction-designer/Connection.vue

```diff
@@ -140,18 +141,16 @@ export default {
 
     this.line = new LeaderLine(start, end, this.options)
 
-    // Add event listeners
-    const self = this
```

Comment:
I remove the reference to self as it did not make any difference in the functionality.  Let me know if there is concern around this change.